.idea/copyright/TerasologyEngine_brief.xml

```diff
@@ -0,0 +1,8 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="allowReplaceRegexp" value="20\d\d" />
+    <option name="keyword" value="Copyright.*Moving\s?Blocks" />
```

Comment:
Do we want to go for "The Terasology Foundation" instead of "Moving Blocks"? @Cervator ?

Reply:
Yes please - good catch 👍 

That's been a global change far down the todo list for a while now